### DISCUSSION TITLE
Crowdsec Collection

### DISCUSSION THREAD

**Discussion by Aurel004** [zitadel/zitadel#8070] [General]
Hi,

I am running crowdsec on my server and I would like to know if it's planned to add a collection (parser and scenario) of Zitadel to Crowdsec ?

As Zitadel becomes the "entrypoint" of security, it would be really nice to detect and ban everywhere

Thank you

**Comment by hifabienne** [comment]
Hi @Aurel004 
At the moment we do not have planned an integration with crowdsec.
This might change in the future, if we do see a significant demand from customers/community.

I don't know enough about crowdsec to be able to tell you if you could already somehow integrate it in a more generic way. e.g using zitadel events/ API.

Maybe you can elaborate a little on what you would expect or what you would need from ZITADEL?
